src/main/java/me/clip/placeholderapi/expansion/manager/LocalExpansionManager.java

```diff
@@ -286,7 +286,7 @@ public boolean register(@NotNull final PlaceholderExpansion expansion) {
 
   @ApiStatus.Internal
   public boolean unregister(@NotNull final PlaceholderExpansion expansion) {
-    if (expansions.remove(expansion.getIdentifier()) == null) {
+    if (expansions.remove(expansion.getIdentifier().toLowerCase()) == null) {
```

Comment:
Just a minor thing, but perhaps use `Locale.ROOT` in `toLowerCase`, just to be safe?

Reply:
https://github.com/PlaceholderAPI/PlaceholderAPI/blob/151fb08db89a094db832e893ddb09c7ada604105/src/main/java/me/clip/placeholderapi/expansion/manager/LocalExpansionManager.java#L196
copied here